cmd/tailscale/cli/ssh.go

```diff
@@ -43,14 +45,28 @@ The 'tailscale ssh' wrapper adds a few things:
   system 'ssh' command that connects via a pipe through tailscaled.
 * It automatically checks the destination server's SSH host key against the
   node's SSH host key as advertised via the Tailscale coordination server.
+
+
+Tailscale can also be integrated with the system 'ssh' and related commands
+by using the --config flag. This will output an SSH config snippet that can 
+be added to your ~/.ssh/config file to enable Tailscale for all SSH connections.
 `),
 	Exec: runSSH,
+	FlagSet: (func() *flag.FlagSet {
+		fs := newFlagSet("ssh")
+		return fs
+	})(),
 }
 
 func runSSH(ctx context.Context, args []string) error {
 	if runtime.GOOS == "darwin" && version.IsMacAppStore() && !envknob.UseWIPCode() {
-		return errors.New("The 'tailscale ssh' subcommand is not available on macOS builds distributed through the App Store or TestFlight.\nInstall the Standalone variant of Tailscale (download it from https://pkgs.tailscale.com), or use the regular 'ssh' client instead.")
+		return errors.New(`
+The 'tailscale ssh' subcommand is not available on macOS builds distributed through the App Store or TestFlight.
+Install the Standalone variant of Tailscale (download it from https://pkgs.tailscale.com).
+Or run tailscale ssh --config >> ~/.ssh/config to use the regular 'ssh' client instead.
+`)
```

Comment:
unrelated change. revert? This also adds a newline at the beginning.

Reply:
I had to add the instructions to output the configuration so it can be appended to the `~/.ssh/config`.

Reply:
`tailscale ssh --config` is not a valid command. Neither is `tailscale configure sshconfig --display` which you reference elsewhere.

Reply:
The `ssh --config` was from a previous iteration I missed updating. The `configure sshconfig --display` is a new command added, I think that got lost with your squashing there was a `cmd/tailscale/cli/configure-ssh.go` file as well that doesn't appear to exist now

Reply:
Whoops, indeed. Fixed.